### PR TITLE
Add admissionwebhook configuration to hpp-operator

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by tiraboschi** [comment]
/hold
Testing it on https://github.com/kubevirt/hyperconverged-cluster-operator/pull/1603 with latest hppo image

File: deploy/webhooks.yaml [hunk 1]
```diff
@@ -152,4 +152,52 @@ webhooks:
...
-  timeoutSeconds: 10
\ No newline at end of file
+  timeoutSeconds: 10
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hostpath-provisioner-operator-webhook-service-cert
+  labels:
+    name: hostpath-provisioner-operator
```
Comment (tiraboschi) [inline deploy/webhooks.yaml:162]:
```suggestion
    name: kubevirt-hyperconverged
```

**Review by tiraboschi — CHANGES_REQUESTED** [review]

File: deploy/webhooks.yaml (hunk 1 above)
Comment (awels) [reply deploy/webhooks.yaml:162]: Same on line 177?

Comment (tiraboschi) [reply deploy/webhooks.yaml:162]: No, I think only here

Comment (awels) [reply deploy/webhooks.yaml:162]: Okay, note the ssp one above doesn't have it
```yaml
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: ssp-operator-service-cert
   labels:
     name: ssp-operator <-------
 spec:
   secretName: ssp-operator-service-cert
   dnsNames:
   - ssp-operator-service.kubevirt-hyperconverged.svc
   issuerRef:
     name: selfsigned

```

**Comment by erkanerol** [comment]
New PR: 1603

**Pull request closed without merging**